src/main/java/uk/gov/hmcts/divorce/caseworker/service/notification/FinalOrderGrantedNotification.java

```diff
@@ -78,4 +113,17 @@ private Map<String, String> solicitorTemplateContent(final CaseData caseData,
 
         return templateVars;
     }
+
+    private Map<String, String> citizenTemplateContent(final CaseData caseData,
+                                                         final Long caseId,
+                                                         final Applicant applicant,
+                                                         final Applicant partner) {
+        Map<String, String> templateVars =
+            commonContent.mainTemplateVars(caseData, caseId, applicant, partner);
+
+        // temporarily set to false, need to update when Final Order switch to sole journey is being developed
```

Comment:
Might be best to add a TODO: tag to this comment

Reply:
updated